src/additional/accept.js

```diff
@@ -14,11 +15,23 @@ $.validator.addMethod("accept", function(value, element, param) {
 		// If we are using a wildcard, make it regex friendly
 		typeParam = typeParam.replace(/\*/g, ".*");
 
+		// Allow checking for BLANK mimetype.
+		if (typeParam.match(/^\|/) || typeParam.match(/\|\|/)) {
```

Comment:
Done!

As for unit tests, I poked around in the tests directory and couldn't find any existing tests of the "accept" method. There are lots of tests for the "extension" method in /test/methods.js, but... I'm not super familiar with this codebase but I thought "accept" was intended to supercede "extension"?